juniper_official/Server-Monitoring/kube-metrics/check-node-status.rule

```diff
@@ -0,0 +1,73 @@
+healthbot {
+    topic kube-metrics {
+        rule check-node-status {
+            keys [ condition node ];
+            synopsis "";
+            description "Checks node status";
```

Comment:
For each node, we track info about NetworkUnavailable, MemoryPressure, DiskPressure, PIDPressure, etc. and ensure that these healthy.

Reply:
Added above description.